src/MonitoringFunctions.Common/TeamsConnector.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace MonitoringFunctions.Common
+{
+    /// <summary>
+    /// Represents a connector for MS Teams.
+    /// </summary>
+    internal sealed class TeamsConnector
+    {
+        private static readonly HttpClient HttpClient = new HttpClient();
+        private const string IncidentCardFormat = "{{\"@context\":\"https://schema.org/extensions\",\"@type\":\"MessageCard\",\"themeColor\":\"FF5555\",\"title\":\"{0}\",\"text\":\"{1}\",\"potentialAction\":[{{\"@type\":\"OpenUri\",\"name\":\"Go To Work Item\",\"targets\":[{{\"os\":\"default\",\"uri\":\"{2}\"}}]}}]}}";
```

Comment:
minor: would it avoid tech dept to instead of creating payload via parameter formatting to instead 
1. create a type, 
1. instantiate it & initialize its' state
1. then serialize it?

Reply:
I think you could also move this definition directly into the SendIncidentCard method and use an anonymous type.

Reply:
Considering my availability this week, I would like to keep this as is at this time.
If we need to make a change next time, we can update as suggested.

Reply:
minor: the "I'll do it next time" bit usually = **tech-debt**.  Unless there is an urgency to finish, please consider following @nohwnd 's advise.